### DISCUSSION TITLE
Is there any way to make Tab draggable?

### DISCUSSION THREAD

**Discussion by hexwit** [mantinedev/mantine#1801] [Q&A] [answered]
Hi, I have a task, to make a tab draggable, to be able to change its position via drag'n'drop. 
In usual case I would use @dnd-kit, but mantine tabs implemented in that way, when tab itself and tab bode are tightly coupled. 
It is not possible to integrate dnd-kit from outside. 
Is there any workaround how to implement this?

**Comment by rtivital** [comment, marked as answer]
Maybe you will be able to implement this with 5.0 update with new Tabs component API – https://v5.mantine.dev/core/tabs/
It will not be supported in 4.x versions.

**Reply by hexwit** [reply]
Thanks, this component structure looks better. Do you know the release date of v5?
